code/datums/traits/negative.dm

```diff
@@ -146,6 +149,8 @@
 
 /datum/quirk/family_heirloom/on_clone(data)
 	heirloom = data
+	var/datum/component/heirloom/C = heirloom.GetComponent(/datum/component/heirloom)
+	C.owner = quirk_holder
```

Comment:
You should use the mind instead of the mob so this doesn't need to be updated

Reply:
I agree, but quirks are stored on the mob level, and copied across during cloning, because I don't know why

Reply:
I'm saying this because I don't want a get component call. I don't care how other systems handle it.